src/drivers/ammo-driver.js

```diff
@@ -1,10 +1,174 @@
-var Driver = require('./driver');
+/* global THREE */
+let Driver = require("./driver");
+window.AmmoModule = window.Ammo;
+window.Ammo = null;
 
-function AmmoDriver () {
-  throw new Error('[AmmoDriver] Driver not implemented.');
+const EPS = 10e-6;
+
+function AmmoDriver() {
+  this.collisionConfiguration = null;
+  this.dispatcher = null;
+  this.broadphase = null;
+  this.solver = null;
+  this.physicsWorld = null;
+  this.debugDrawer = null;
+
+  this.els = {};
+  this.eventListeners = [];
+  this.collisions = {};
 }
 
 AmmoDriver.prototype = new Driver();
 AmmoDriver.prototype.constructor = AmmoDriver;
 
 module.exports = AmmoDriver;
+
+/* @param {object} worldConfig */
+AmmoDriver.prototype.init = function(worldConfig) {
+  return new Promise(resolve => {
+    AmmoModule().then(result => {
```

Comment:
Could we remove one layer of wrapper here?

```
return AmmoModule().then((Ammo) => {
 // ...
});
```

If there's something weird going on like AmmoModule returning a 'fake' Promise, let's just add a comment to explain that.

Reply:
Correct, Emscripten doesn't use real promises. I'll add a comment.